automation/check-patch.openshift-3.10-multus-release.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
```

Comment:
t's the same as in kubernetes (and maybe other providers). Could we just use symlink?